packages/chart/src/components/EditorPanel/EditorPanel.tsx

```diff
@@ -3704,7 +3704,10 @@ const EditorPanel = () => {
                     </>
                   } */}
                   <Select
-                    display={config.series?.length > 1}
+                    display={
+                      ![undefined, '- Select - '].includes(config.series?.[0].dynamicCategory) ||
+                      config.series?.length > 1
+                    }
```

Comment:
Can you create two variables:

const hasDynamicCategory = ![undefined, '- Select - '].includes(config.series?.[0].dynamicCategory)
const hasMultipleSeries = config.series?.length > 1

and then change the display to be display={hasDynamicCategory || hasMultipleSeries}